pkg/plugins/resources/k8s/plugin.go

```diff
@@ -24,9 +23,6 @@ func (p *plugin) NewResourceStore(pc core_plugins.PluginContext, _ core_plugins.
 	if !ok {
 		return nil, errors.Errorf("k8s controller runtime Manager hasn't been configured")
 	}
-	if err := mesh_k8s.AddToScheme(mgr.GetScheme()); err != nil {
-		return nil, errors.Wrap(err, "could not add to scheme")
-	}
```

Comment:
it looks like integration tests are failing because of this removal, what is the reason of doing it?